src/User/UserContainer.jsx

```diff
@@ -1,47 +1,53 @@
 import React, { useState, useEffect } from "react";
 import { Switch, Route, useParams } from "react-router-dom";
-import { withCookies, useCookies } from "react-cookie";
+import {  useCookies } from "react-cookie";
 import { cookieName } from "../Constants/Cookies";
-import { getUserById } from "../api.js";
 import ProfilePage from "./ProfilePage";
 import UpdateProfile from "./UpdateProfile";
+import { getUserById } from "../api.js";
 
 const UserContainer = () => {
-  const [user, setUser] = useState(null);
-  const [isLoading, setIsLoading] = useState(true);
-  const [error, setError] = useState(false);
-  const { id } = useParams();
-  const [cookies] = useCookies([cookieName]);
+	const [user, setUser] = useState(null);
+	const [isLoading, setIsLoading] = useState(true);
+	const [error, setError] = useState(false);
+	const { id } = useParams();
+	const [cookies] = useCookies([cookieName]);
 
-  const fetchData = async userId => {
-    setIsLoading(true);
+	const fetchData = async () => {
+		setIsLoading(true);
 
-    try {
-      const user = await getUserById(userId, cookies.sessionToken.token);
-      setUser(user);
-    } catch (error) {
-      setError(true);
-    }
-    setIsLoading(false);
-  };
+		try {
+			const user = await getUserById(id, cookies.sessionToken.token);
```

Comment:
Is there any point where this sessionToken doesn't exist, Can you get to this page when logged out i can't remember?

Reply:
Only logged in members should be able to get to this page!